src/callAPIMethod.js

```diff
@@ -26,13 +26,16 @@ export default function callAPI(
 
     return fetch(url, {...defaults.fetchOptions, ...fetchOptions, ...options})
         .then( result => {
-            const promiseCallback = result.body ? result[method]() : new Promise(resolve => resolve(result.body));
+            const promiseCallback = result[method]();
```

Comment:
>  I think the issue is with isomorphic-fetch not being really isomorphic...

Isomorphic-fetch implements the minimal functionality of fetch.

> Did you try using Response.blob() instead?

I didn't try to do it. But I think that the transform data from Blob to the right data type will be problematic. It requires additional conditions. 

I'll try to do something in this way.